src/vivarium_gates_mncnh/data/lbwsg_paf_branches.yaml

```diff
@@ -1,7 +1,7 @@
-input_draw_count: 250
+input_draw_count: 10
 random_seed_count: 1
 
-#input_draws: [115, 60, 368, 197, 79, 244, 272, 167, 146, 71] 
+input_draws: [115, 60, 118, 197, 79, 244, 22, 167, 146, 71]
```

Comment:
Sorry, I've been in a bit of an edit war with @hussain-jafari on this -- but I think it is necessary for now because [VCT hardcodes the number of draws](https://github.com/ihmeuw/vivarium_cluster_tools/blob/9d893e082a840270e88c1ea2e68d86fc28ff91f3/src/vivarium_cluster_tools/psimulate/branches.py#L21), so when you ask it to select 250 draws, it selects some >=250 (which don't exist in GBD 2023).

@albrja is going to make a JIRA ticket about updating VCT.